src/app/components/register/register.component.ts

```diff
@@ -50,7 +50,7 @@ export class RegisterComponent implements OnInit {
       this.usernameError = 'Username must be at least 1 character';
       this.usernameStatus = false;
     } else if (this.myValidateFunction(this.username) === false) {
-      this.usernameError = 'Username can not contain any special character';
+      this.usernameError = 'Username can not contain special character';
```

Comment:
characters

Reply:
corrected and solved the issue because adding 1 more character (1 "s") made the error message two 2 lines on the smallest mobile view